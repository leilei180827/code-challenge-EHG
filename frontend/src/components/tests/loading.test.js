import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../loading';

test('loading page render correctly', () => {
  const component = renderer.create(
    <Loading></Loading>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});