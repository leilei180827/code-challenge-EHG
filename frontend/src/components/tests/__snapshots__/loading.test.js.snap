// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading page render correctly 1`] = `
<div
  className="lds-dual-ring"
  data-testid="loading"
/>
`;
